modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/build.gradle

```diff
@@ -4,8 +4,8 @@ import com.liferay.gradle.util.StringUtil
 task cleanSidecar
 task deploySidecar(type: Copy)
 
-String elasticsearchClientVersion = "7.17.26"
-String luceneVersion = "8.11.3"
+String elasticsearchClientVersion = "7.17.26" //the client should be also updated? or its not related (sidecar in 8 version and client on a previous version can cause problems?)
+String luceneVersion = "8.11.3" //we should update the lucene version? the elasticsearch 8.17.3 use the 9.12.0 of lucene
```

Comment:
The client should be fine as is, for now. We are using the REST compatibility mode of Elasticsearch 8 by sending specific headers with each requests that instruct Elasticsearch 8 to "behave as as Elasticsearch 7".

We're going to implement a native Elasticsearch 8 connector in M2.

Reply:
The Lucene version ia also corresponding to the client implementation afaik, so it can probably also stay.

What you could do however is to bump the client and related artifacts to 7.17.27 or whatever is the latest micro release available for 7.17.